src/actions/index.test.js

```diff
@@ -89,4 +111,256 @@ describe('Actions', () => {
+    const state = { key: 'value' };
+    const subnetworkId = 43;
+
+    beforeEach(() => {
+      setBelief(nodeId, state, subnetworkId)(dispatch);
+    });
+
+    it('calls dispatch with type SET_BELIEF and payload', () => {
+      expect(dispatch).toHaveBeenNthCalledWith(1, {
+        type: SET_BELIEF,
+        payload: { nodeId, state, subnetworkId },
+      });
+    });
+
+    it('calls dispatch persist action', () => {
+      expect(dispatch).toHaveBeenNthCalledWith(2, persistStateAction);
+    });
+  });
+
+  describe('changeNetworkProperty', () => {
+    const name = 'name';
+    const value = 'value';
+
+    beforeEach(() => {
+      changeNetworkProperty(name, value)(dispatch);
+    });
+
+    it('calls dispatch with type CHANGE_NETWORK_PROPERTY and payload', () => {
+      expect(dispatch).toHaveBeenNthCalledWith(1, {
+        type: CHANGE_NETWORK_PROPERTY,
+        payload: { name, value },
+      });
+    });
+
+    it('calls dispatch persist action', () => {
+      expect(dispatch).toHaveBeenNthCalledWith(2, persistStateAction);
+    });
+  });
+
+  describe('changeNodeId', () => {
+    const id = 42;
+    const nextId = 43;
+
+    beforeEach(() => {
+      changeNodeId(id, nextId)(dispatch);
+    });
+
+    it('calls dispatch with type CHANGE_NODE_ID and payload', () => {
+      expect(dispatch).toHaveBeenNthCalledWith(1, {
+        type: CHANGE_NODE_ID,
+        payload: { id, nextId },
+      });
+    });
+
+    it('calls dispatch persist action', () => {
+      expect(dispatch).toHaveBeenNthCalledWith(2, persistStateAction);
+    });
+  });
+
+  describe('changeNodeDescription', () => {
+    const id = 42;
+    const description = 'lorem ipsum';
+
+    beforeEach(() => {
+      changeNodeDescription(id, description)(dispatch);
+    });
+
+    it('calls dispatch with type CHANGE_NODE_DESCRIPTION and payload', () => {
+      expect(dispatch).toHaveBeenNthCalledWith(1, {
+        type: CHANGE_NODE_DESCRIPTION,
+        payload: { id, description },
+      });
+    });
+
+    it('calls dispatch persist action', () => {
+      expect(dispatch).toHaveBeenNthCalledWith(2, persistStateAction);
+    });
+  });
+
+  describe('changeNodePosition', () => {
+    const id = 42;
+    const x = 0;
+    const y = 0;
+
+    beforeEach(() => {
+      changeNodePosition(id, x, y)(dispatch);
+    });
+
+    it('calls dispatch with type CHANGE_NODE_POSITION and payload', () => {
+      expect(dispatch).toHaveBeenNthCalledWith(1, {
+        type: CHANGE_NODE_POSITION,
+        payload: { id, x, y },
+      });
+    });
+
+    it('calls dispatch persist action', () => {
+      expect(dispatch).toHaveBeenNthCalledWith(2, persistStateAction);
+    });
+  });
+
+
+  describe('addNode', () => {
+    const id = 42;
+    const states = ['fake-statuses'];
+    const position = 42;
+
+    beforeEach(() => {
+      addNode(id, states, position)(dispatch);
+    });
+
+    it('calls dispatch with type ADD_NODE and payload with payload', () => {
```

Comment:
`calls dispatch with type ADD_NODE and payload` 💅

Reply:
Ops.. it'was a typo :D
I'll fix it